### DISCUSSION TITLE
Loading state for checkbox

### DISCUSSION THREAD

**Discussion by Whidix** [carbon-design-system/carbon-components-svelte#1858] [Ideas]
Hello,
I have a dashboard to manage user's permissions, when I check/uncheck a permission a fetch request is made but during the request I would like an animation. Since I use checkboxs with no label, would it be possible to have a different skeleton when labels are disabled, like the loading.

**Comment by metonym** [comment]
The default Checkbox skeleton serves as a loading state, and does not have a label: https://svelte.carbondesignsystem.com/components/Checkbox#skeleton
